ui/apps/platform/src/Containers/ComplianceEnhanced/Schedules/hooks/useWatchLastSnapshotForComplianceReports.ts

```diff
@@ -57,7 +57,13 @@ function useWatchLastSnapshotForComplianceReports(
             Promise.allSettled(configurations.map(({ id }) => fetchLastJobForConfiguration(id)))
                 .then((snapshotResults) => {
                     const result: ComplianceReportSnapshotLookup = configurations.reduce(
-                        (acc, { id }, index) => ({ ...acc, [id]: snapshotResults[index] }),
+                        (acc, { id }, index) => {
+                            const result = snapshotResults[index];
```

Comment:
For what it is worth, here is only previous occurrence of `allSettled` in components:
https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/src/Containers/Compliance/Dashboard/ManageStandardsModal.tsx#L43-L67

It looks like the modal renders just the first error.
* To **save** changes, the count of fulfilled versus rejected was important information.
* To **display** status, is it possible to display one error in warning `Alert` on the page and concise indicator in each table cell for which status in unavailable?

P.S. There is one other occurrence in NetworkService.js file.

Reply:
This is a good question to discuss 🤔 . My initial thought is that this is probably workflow dependent, and could change based on what the user can control.

If I understand correctly, an error here will not be due to anything the user has control over. Either there is a network flake, a central restart, some mismatch between config ids and the data on the server, or simply a programming error edge case we didn't catch.

Since there isn't anything actionable the user can do, I feel like showing an error in the UI might be a little jarring, especially if the error is a fluke transient thing (network error, central restart), in which case I think how you have it now might be better than displaying an error for a couple seconds, and then having it disappear.

In the more persistent cases - programming error, data mismatch, etc, I'm still not sure displaying an error message to the user would be helpful. We should definitely `Raven` capture this information though so it is available in a diag bundle in the case where a user submits a support ticket due to consistent missing snapshot information.

Reply:
Thanks for the suggestions. I started the discussion on `#team-acs-ux` if you'd like to continue it there.

https://redhat-internal.slack.com/archives/C02MN2N2UG4/p1727978525138319. 

I was and still am leaning towards not displaying it in the UI but capturing it with `Raven`. We should already be doing that in the `fetchComplianceReportHistory` service function.